tests/cases/struct/LV_NeoHookean_passive_genBC/svFSIplus.xml

```diff
@@ -59,9 +59,6 @@
    <Constitutive_model type="neoHookean"> 
       </Constitutive_model> 
    <Dilational_penalty_model> ST91 </Dilational_penalty_model>
-   <Viscosity model="Constant" >
-         <Value> 0.0 </Value>
-      </Viscosity>
 
```

Comment:
I would leave this in to show users that there is a `Viscosity` parameter and that its value is 0.

Reply:
@aabrown100-git the `Solid_viscosity` parameter is only used for CMM.

Reply:
Currently, I don't think `Solid_viscosity` is used for CMM. In [cmm.cpp](https://github.com/SimVascular/svFSIplus/blob/main/Code/Source/svFSI/cmm.cpp), I don't see any reference to `Solid_viscosity`. On the otherhand, `Solid_viscosity` appears to be used to set the viscosity value in `struct` [see here](https://github.com/SimVascular/svFSIplus/blob/b2271ee8b8ec2ded8e0852230162817e828fd700/Code/Source/svFSI/sv_struct.cpp#L619) and `ustruct` [see here](https://github.com/SimVascular/svFSIplus/blob/b2271ee8b8ec2ded8e0852230162817e828fd700/Code/Source/svFSI/ustruct.cpp#L1771).

Reply:
@ktbolt When you get a chance, could you comment how you suggest resolving this issue? BTW, I have another [issue](https://github.com/SimVascular/svFSIplus/issues/273) to reorganize the solid viscosity implementation for struct and ustruct.

Reply:
Thanks @ktbolt for these suggestions. Can I suggest merging this pull request, just so that the `struct`, `ustruct`, and `fsi` test reference solutions are properly updated with zero viscosity?

Then, we can continue discussing reorganizing the viscosity implementation [here](https://github.com/SimVascular/svFSIplus/issues/273#issuecomment-2375327612)

Reply:
@aabrown100-git Let's merge!